autogluon/inference/docker/0.3/py3/Dockerfile.cpu

```diff
@@ -11,13 +11,24 @@ LABEL maintainer="Amazon AI"
 LABEL dlc_major_version="1"
 
 ARG PYTHON_VERSION=3.7.10
+ARG OPENSSL_VERSION=1.1.1l
 
 RUN apt-get update \
  && apt-get -y upgrade \
  && apt-get autoremove -y \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
+# install OpenSSL
```

Comment:
Is there a reason we can't install openSSL from apt-get?

Reply:
ditto